notify/pushover.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
fix the shebang same at others

Reply:
updated to match the others, No idea where I got the other one from

Reply:
why not read another hook first ?

https://github.com/Neilpang/acme.sh/blob/master/notify/mail.sh#L1